app/client.ts

```diff
@@ -0,0 +1,47 @@
+import { PrismaClient } from "@prisma/client";
+
+const prisma = new PrismaClient();
+
+export interface Report {
+  reporterFid: bigint;
+  sybilFid: bigint;
+  castHash: string | undefined;
+  messageHash: string | undefined;
+  network: number | undefined;
+  reportTimestamp: string | undefined;
+}
+
+export async function createReport(report: Report) {
+  const _report = await prisma.report.create({
+    data: report,
```

Comment:
the `_variable` is pattern commonly used for variables that are not used, it would change to
```js
export async function createReport(data: Report) {
  const report = await prisma.report.create({
    data,
  });
  return report;
}
```